docs/setup/node/options.md

```diff
@@ -0,0 +1,22 @@
+# Setup Options
+
+The easyest way to setup the DSK Node is through the Azure Marketplace. Yet other setup options are also available. 
```

Comment:
I would propose: Although other setup options are also available, the easiest way to setup the DSK Node is through the Azure Marketplace.